cfgov/v1/models/atoms.py

```diff
@@ -1,6 +1,70 @@
+from django.core.exceptions import ValidationError
+
 from wagtail.wagtailcore import blocks
+from wagtail.wagtailimages.blocks import ImageChooserBlock
+
+
+def isRequired(field_name):
+    return [str(field_name) + ' is required.']
 
 
 class Hyperlink(blocks.StructBlock):
-    label = blocks.CharBlock(max_length=50)
-    href = blocks.CharBlock(default='/')
+    text = blocks.CharBlock(max_length=50, required=False)
```

Comment:
ok, so the problem is that `value.slug` should be `value.text`. I am referring to https://github.com/cfpb/cfgov-refresh/pull/1154/files#diff-3f1f0f338f0a28057bf1dac682d9dca2R36.